airbyte-webapp/src/utils/testutils.tsx

```diff
@@ -47,7 +47,7 @@ export async function render<
 }
 export const TestWrapper: React.FC = ({ children }) => (
   <ThemeProvider theme={{}}>
-    <IntlProvider locale="en" messages={en}>
+    <IntlProvider locale="en" messages={en} onError={() => null}>
```

Comment:
Maybe do a `console.error` instead?

Reply:
This is used during testing, and the purpose I added this there is to get rid of the console errors we're currently seeing when testing a component that uses strings from the cloud package, which are not actually loaded here (since it's OSS code). I think it's fine to just ignore this here.